community-stabilization-fund-api/pages/api/package-items/index.ts

```diff
@@ -0,0 +1,72 @@
+import { PrismaClient } from '@prisma/client';
+
+import type { PackageItemDTO } from '../../../src/db';
+
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+const prisma = new PrismaClient({
+  datasources: { db: { url: process.env.DATABASE_URL } },
+});
+
+const PackageItemHandler = (req: NextApiRequest, res: NextApiResponse) => {
+  const { method, body, query } = req;
+  switch (method) {
+    case 'GET':
+      getAllPackageItems(res, query);
+      break;
+    case 'POST':
+      createPackageItem(body, res);
+      break;
+    case 'DELETE':
+      deletePackageItems(body, res);
+      break;
+    default:
+      res.setHeader('Allow', ['GET', 'POST', 'DELETE']);
+      res.status(405).end(`Method ${method} Not Allowed`);
+      break;
+  }
+};
+
+const createPackageItem = async (body: any, res: NextApiResponse) => {
+  try {
+    const result = await prisma.package_item.create({ data: body });
+    return res
+      .status(201)
+      .send('Successfully created question with id: ' + result.id);
+  } catch (error) {
+    console.error({ error });
+    throw error;
+  }
+};
+
+const getAllPackageItems = async (res: NextApiResponse, query: any) => {
```

Comment:
We're not taking advantage of using a join table on Prisma if we're not using 'include: table_name' to get the related items. In both packageItem and packageGroup we gotta get their related package_group_items.